### ISSUE TITLE
Add about Wildlife & Conservation

### ISSUE THREAD

**Issue by Aditijainnn** [ajaynegi45/Uttarakhand-Culture-NewUI#61]
### Is this feature already requested?

- [X] I have checked "open" and "closed" issues, and this is not a duplicate.

### Problem or Missing Functionality

Adding about Wildlife Sanctuaries & National Parks:

### Feature Description

 Feature Jim Corbett National Park, Rajaji National Park, and the Binsar Wildlife Sanctuary with interactive features like virtual safaris, animal tracking apps, and photo galleries for wildlife enthusiasts.

please assign this to me under gssoc ext and hacktoberfest

### Would you like to work on this feature?

Yes

**Comment by ajaynegi45** [comment]
> ### Is this feature already requested?
> * [x]  I have checked "open" and "closed" issues, and this is not a duplicate.
> 
> ### Problem or Missing Functionality
> Adding about Wildlife Sanctuaries & National Parks:
> 
> ### Feature Description
> Feature Jim Corbett National Park, Rajaji National Park, and the Binsar Wildlife Sanctuary with interactive features like virtual safaris, animal tracking apps, and photo galleries for wildlife enthusiasts.
> 
> please assign this to me under gssoc ext and hacktoberfest
> 
> ### Screenshots
> _No response_
> 
> ### Would you like to work on this feature?
> Yes
> 
> ### Implementation Plan
> _No response_

Hey @Aditijainnn,

Thanks for showing interest in tackling this issue! I'd be happy to assign it to you, but before I do, could you walk me through how you plan to implement this feature in detail?

Also, before jumping in, I recommend giving the [[Readme.md](https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI/blob/main/README.md)](https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI/blob/main/README.md) a thorough read. It'll help you get a solid grasp of the project's goals and purpose. Plus, check out the [[Contributing.md](https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI/blob/main/contributing.md)](https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI/blob/main/contributing.md) file for some handy guidelines that’ll make your contribution process smoother, especially if you're new around here.

If you run into any questions or need help along the way, don’t hesitate to reach out!

**Comment by AgarwalSneh** [comment]
Can I work on it?  Different kinds of wildlife section will be provided, when the user clicks on a particular section  a new page will open up and the address will be provided with the description.

**Comment by ajaynegi45** [comment]
> Can I work on it? Different kinds of wildlife section will be provided, when the user clicks on a particular section a new page will open up and the address will be provided with the description.

Feel free to start working, and if you have any questions or need assistance during the process, please don't hesitate to reach out.

**Comment by AgarwalSneh** [comment]
Sure, thank you

**Comment by Adrija05** [comment]
<img width="914" alt="Screenshot 2024-10-05 at 11 16 54 PM" src="https://github.com/user-attachments/assets/791fa9bf-5558-4f81-9e8c-0afc27e89c5d">

**Comment by ajaynegi45** [comment]
Hi @AgarwalSneh,

Hope you're doing great! 😊 Just wanted to check in and see how things are coming along with the issue you're working on. Any progress or updates you’d like to share? And if you’ve run into any roadblocks or need a hand with anything, please don’t hesitate to reach out!

Looking forward to hearing from you!

**Comment by AgarwalSneh** [comment]
Yes Sir, will complete in a few days and will show you the project soon.
